packages/server/package.json

```diff
@@ -89,7 +89,6 @@
     "@commitlint/config-conventional": "^11.0.0",
     "@typescript-eslint/eslint-plugin": "^5.26.0",
     "@typescript-eslint/parser": "^5.26.0",
-    "api": "^4.2.1",
```

Comment:
This was the one dependency not supported in Node 18. I did some research and I think it is safe to delete. It was originally used in the old arpa repository in a script that copied Render configurations. Since we're not using Render anymore and since that script was not copied over when arpa was moved to this repository, I think this dependency is safe to delete.